iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -88,16 +52,16 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager) {
       Shape::createExpandFunctionDynamicDimsPass());
 
   // Special case peephole optimizations.
-  if (clEnable1x1ConvToMatmul) {
```

Comment:
Out of curiosity do you think in the future these could be moved to a new pipeline that sat before flow? They don't seem to be flow related and it may make it easier to upstream them to have them operating strictly on linalg (unless the dynamic dimension expansion is load-bearing here).

Reply:
yeah, thats a good point. Though some of these should go away all together, the rest might be better off in some input pipeline. I'd say that is TBD, we probably dont want "many" input pipelines in IREE propoer. Ideally the front ends just generate the code that starts with Flow?